src/containers/BirdsBlock/BirdsBlock.tsx

```diff
@@ -0,0 +1,101 @@
+import styles from './BirdsBlock.module.scss';
+
+import { Bird } from '@containers/Bird/Bird';
+import { DescriptionBirdDefault } from '@view/DescriptionBirdDefault/DescriptionBirdDefault';
+import { MouseEventHandler, useState } from 'react';
+import { ButtonNavigation } from '@view/ButtonNavigation/ButtonNavigation';
+import { DescriptionBird } from '@view/DescriptionBird/DescriptionBird';
+
+import { ADD_POINTS } from '@constans/reducerActionType';
+import { useDispatch } from 'react-redux';
+import { IBird } from '@constans/interface/birds.interface';
+import { IPropsValueBirds } from './BirdsBlock.interface';
+
+import useSound from 'use-sound';
+import alarmRight from '@static/alarmValue/right.mp3';
+import alarmError from '@static/alarmValue/error.mp3';
+
+const BirdsBlock: React.FC<IPropsValueBirds> = ({ birds, url, setMainBird, setPercentage, percentage }) => {
+  const [bird, addBird] = useState<IBird[]>([]);
```

Comment:
```suggestion
  const [bird, addBird] = useState<IBird | null>(null);
```

На сколько я вижу у тебя тут всегда одна птица. Незачем хранить ее в массиве